plugins/telescope/default.nix

```diff
@@ -112,6 +112,9 @@ helpers.neovim-plugin.mkNeovimPlugin config {
         require('telescope').load_extension(extension)
       end
     '';
+
+    # planets picker requires files in data/memes/planets
+    performance.combinePlugins.pathsToLink = [ "/data/memes/planets" ];
```

Comment:
Having to define this explicitly feels fragile... What would happen if we add another plugin module but don't notice we need to define addition paths to link?

Reply:
> What would happen if we add another plugin module but don't notice we need to define addition paths to link?

Additional paths will be missing from user runtime if `combinePlugins` is enabled, so it depends on what files are missing. For telescope here, it pretty much doesn't matter, because planets is a picker that is only for testing. Most plugins doesn't need any paths except standard.

I think in a description of this options should be clear, that this option is experimental, doesn't give any guaranties etc. So if it works for user's config, he'll get a performance boost, if not — sorry. I've added EXPERIMENTAL to the description for that purpose. But English is not native for me, if you have better description, we can update it.

Reply:
I think the description is good. I think we can support this feature with less guarantees on breaking changes, i.e we won't add tests for _all_ the plugins trying to enable the option, as this might be a too large maintenance burden, but we'll gladly include workarounds if people suggest them